docusaurus.config.js

```diff
@@ -43,7 +43,7 @@ const config = {
         docs: {
           editUrl: "https://github.com/LearningTypeScript/projects/tree/main/",
           include: ["*/*.md", "*/*/*.md"],
-          path: "src/content/external/projects",
+          path: "projects",
```

Comment:
Hmm, I get this error when I try to `yarn start` locally:

```
[INFO] Starting the development server...
[ERROR] Error: The docs folder does not exist for version "current". A docs folder is expected to be found at projects.
    at getVersionMetadataPaths (/home/josh/repos/site/node_modules/@docusaurus/plugin-content-docs/lib/versions/files.js:125:15)
    at async createVersionMetadata (/home/josh/repos/site/node_modules/@docusaurus/plugin-content-docs/lib/versions/index.js:94:68)
    at async Promise.all (index 0)
    at async readVersionsMetadata (/home/josh/repos/site/node_modules/@docusaurus/plugin-content-docs/lib/versions/index.js:146:30)
    at async Object.pluginContentDocs [as plugin] (/home/josh/repos/site/node_modules/@docusaurus/plugin-content-docs/lib/index.js:32:30)
    at async initializePlugin (/home/josh/repos/site/node_modules/@docusaurus/core/lib/server/plugins/init.js:83:32)
    at async Promise.all (index 0)
    at async initPlugins (/home/josh/repos/site/node_modules/@docusaurus/core/lib/server/plugins/init.js:91:21)
    at async loadPlugins (/home/josh/repos/site/node_modules/@docusaurus/core/lib/server/plugins/index.js:26:21)
    at async load (/home/josh/repos/site/node_modules/@docusaurus/core/lib/server/index.js:77:58)
    at async start (/home/josh/repos/site/node_modules/@docusaurus/core/lib/commands/start.js:41:19)
error Command failed with exit code 1.
```

Are you able to run this PR?

Reply:
@JoshuaKGoldberg I tried to run using `yarn start` but was having the same error. 
I am new to working with `docusaurus`. 
Also, I checked the documentation of `docusaurus`.  In the docs versioning, they mentioned having a `docs/` directory. But I can not find any directory named `docs` in this repository. 

Can you enlighten how to fix this up. I tried to google with the error, could not find much resource.

Reply:
> checked the documentation ... mentioned having a `docs/` directory

Could I suggest doing a deeper read of the docs, and trying out making a Docusaurus site or two of your own to try it out? This project configures docs using a pretty standard `docusaurus.config.js` line: https://github.com/LearningTypeScript/site/blob/2f8e8863d5fde61500c5c642b84ef5312f13e583/docusaurus.config.js#L43

This issue around configuring Docusaurus might not be a _good first issue_ for you if you're not familiar with Docusaurus. 🙂